components/chatbot/actions/create-user/create-user.mjs

```diff
@@ -0,0 +1,59 @@
+import chatbot from "../../chatbot.app.mjs";
+
+export default {
+  key: "chatbot-create-user",
+  name: "Create User",
+  description: "Creates new user. [See docs here](https://www.chatbot.com/docs/users/#list-users)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    chatbot,
+    userId: {
+      label: "User ID",
+      description: "The user ID",
+      type: "string",
+    },
+    email: {
+      label: "Email",
+      description: "The email of the user",
+      type: "string",
+      optional: true,
+    },
```

Comment:
> Note: userId is not required when attributes.default_email is provided
https://www.chatbot.com/docs/users/#create-user

It seems that if `email` is supplied, `userId` is not necessary. I think having the `email` prop as `required` instead of `userId` would be better